### ISSUE TITLE
Remove from workspaces works only after doing it twice.

### ISSUE THREAD

**Issue by naryad** [LightTable/LightTable#1934]
Mac OS X 10.9.5, LT 0.7.2
First time I click on "remove from workspace" nothing happens, it corrects removes folder from workspace only on second attempt.

**Comment by kenny-evitt** [comment]
@naryad Please try to reproduce this with the *atom-shell* branch. That's effectively the pre-release for the next major version.

**Comment by BenjaminVanRyseghem** [comment]
I can't reproduce it on the `atom-branch`
but I am not using the same OSX version
(and I can't remember having this issue before)

**Comment by kenny-evitt** [comment]
@naryad If you're going to test a source build, just use *master* – *atom-shell* was merged.

**Comment by naryad** [comment]
@kenny-evitt Installed LightTable from master branch and I can reproduce it with latest code. OS X version used 10.9.2.

**Comment by kenny-evitt** [comment]
@naryad Thanks for reproducing with the latest code. Given that a workaround is readily available, and somewhat documented via this issue, this is going to be a low priority for a (long) while. But pull requests are welcome! If you can look at this, and you having any questions, just ping me in this issue or in [the Gitter (chat) room](https://gitter.im/LightTable/LightTable).
